src/ansys/mapdl/core/pool.py

```diff
@@ -670,7 +677,7 @@ def _spawn_mapdl(
         self._instances[index] = launch_mapdl(
             run_location=run_location,
             port=port,
-            override=self._override,
+            override=True,
```

Comment:
Is there a reason not to use `self._override` anymore or is it a typo?

Reply:
Because we want to respawn an instance in the same directory as the previous one... so it "replace" it properly. If we change the directory, we might lose access to uploaded files or previous results.